README.md

```diff
@@ -34,14 +36,25 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+The difference is, that a responsive website will automatically adapt to a users screen size and device and has kind of a liquid design
```

Comment:
Beautiful!

http://mediumwell.com/responsive-adaptive-mobile/
and a lot of other online resources can furnish you with more differences